indexer/index-emojis.js

```diff
@@ -43,7 +43,14 @@ function updateGithub() {
 }
 
 async function processBatch(inscriptions) {
-    const textInscriptions = inscriptions.filter(it => it.content_type === 'text/plain;charset=utf-8')
+    const allowedContentTypes = [
+        'text/plain;charset=utf-8',
+        'text/plain',
+        'application/json',
```

Comment:
should json be in there? i wonder if there are any, because a plaintext emoji would not be valid json anyway

Reply:
Great call. I added it for safety, but yeah, it doesn't make sense. Removing it.